notebooks-html/bias_random_vars.html

```diff
@@ -54,20 +54,22 @@
 </pre></div></div></div></div></div>
 <div class="cell border-box-sizing text_cell rendered"><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
-<h2 id="Expectation-and-Variance">Expectation and Variance<a class="anchor-link" href="#Expectation-and-Variance">&#182;</a></h2><p>Almost all real-world phenomena contain some degree of randomness, making data generation and collection inherently random processes. Since we fit our models on this data, our models also contain randomness. To represent these random processes mathematically, we use random variables.</p></div></div></div>
+<h2 id="Expectation-and-Variance">Expectation and Variance<a class="anchor-link" href="#Expectation-and-Variance">&#182;</a></h2><p>Almost all real-world phenomena contain some degree of randomness, making data generation and collection inherently random processes. Since we fit our models on these data, our models also contain randomness. To represent these random processes mathematically, we use random variables.</p></div></div></div>
 <div class="cell border-box-sizing text_cell rendered"><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
 <h3 id="Random-Variables">Random Variables<a class="anchor-link" href="#Random-Variables">&#182;</a></h3><p>A <strong>random variable</strong> is an algebraic variable that represents a numerical value determined by a probabilistic event. We typically use capital letters like $ X $ or $ Y $ to denote a random variable.</p>
-<p>We must always specify what a given random variable represents. For example, we may write that the random variable $ X $ represents the number of heads in 10 coin flips. From the definition of a random variable we must be able to determine the possible values that the variable can take on. In this example, $ X $ may only take on values between $ 0 $ and $ 10 $.</p>
+<p>We must always specify what a given random variable represents. For example, we may write that the random variable $ X $ represents the number of heads in 10 coin flips. The definition of a random variable determines the values it can take on. In this example, $ X $ may only take on values between $ 0 $ and $ 10 $.</p>
```

Comment:
I like this rewording :).

Reply:
Thanks! I believe this was Daniel's suggestion, so give credit to him instead.